Microsoft.Toolkit.Uwp.Connectivity/BluetoothLEHelper/BluetoothLEHelper.cs

```diff
@@ -197,7 +197,7 @@ private async Task Init()
         /// <param name="args">The advertisement.</param>
         private async void AdvertisementWatcher_Received(BluetoothLEAdvertisementWatcher sender, BluetoothLEAdvertisementReceivedEventArgs args)
         {
-            await DispatcherQueue.ExecuteOnUIThreadAsync(
+            await DispatcherQueue.EnqueueAsync(
```

Comment:
We had kept these methods with the same name as the DispatcherHelper to help with folks migrating code from Dispatcher. Not sure if we want to change it? Or is it easy for us to have a VS analyzer thing to help people by making a suggestion to change our old thing to the new thing?

@azchohfi thoughts?

Reply:
We could create a method with the same old name that is deprecated, just to help migration, and suggest the change to the new method name.

Reply:
I'm slightly confused about the migration - isn't this class new in the 7.0 release? As in, wouldn't it still be a breaking change even keeping the original method names, as users would still have to apply the extension on a `DispatcherQueue` instead of the previous `CoreDispatcher`? My rationale here was that if we were going to break anyway, we could just throw in an API rename too in the process, as all consumers would have to manually change all those lines of code anyway? 🤔

Anyway let me know how you'd like me to proceed here, I can definitely add those methods too if you think it's best!

Reply:
What would be the point of changing the name? Is there any new functionality added from one to the other? Just a name change is an awful reason to deprecate something. If this hasn't been released yet, then change away!

Reply:
@skendrot The reasons for the name change were basically that:

- `EnqueueAsync` is consistent with the built-in `TryEnqueue` method of the `DispatcherQueue` class.
- It's also more explicit about failures as it follows the standard `Do/TryDo` pattern in the BCL - in this case `TryEnqueue` returns a `bool` indicating whether it was successful, whereas this will just return an exception, hence the name.
- It is more representative of what it's doing, as this method isn't related to "the" UI thread per se, but it's only working on a given `DispatcherQueue`. As in, you could have multiple windows, each with a different queue, and you'd be effectively just enqueueing an action on one of them, not just running it on "the" UI thread (there isn't one).

And yeah I went with the name change in this specific case just because the original `DispatcherHelper` class was already made obsolete in 7.0, and this whole class would be new in the next toolkit release 🙂

Reply:
You are right. I thought we shipped this already, but we didn't.
Lets just make sure that the old methods point to the new ones thru the deprecation messages, so the migration is easy.